.gitignore

```diff
@@ -24,3 +24,7 @@ ExportedObj/
 
 # Unity3D Generated File On Crash Reports
 sysinfo.txt
+*.meta
```

Comment:
We need to keep the meta file or the unity will break

Reply:
ok...ignr it